### ISSUE TITLE
How to change text color of 'Tip of the day' and the Notifications.

### ISSUE THREAD

**Issue by DrDoa** [ChrisRM/material-theme-jetbrains#841]
### Summary
Thanks for the new version of Material UI Plugins which enables modifying the ide colors.
But when I am  making customization to my ide,  I find that the text color of ''Tip of the day' and 'Notifications' cannot be change. so if it possible to change it ?
...

### Expected behavior
An interface to change the text color of ''Tip of the day' and 'Notifications' 
...

### Actual behavior
Cannot find interface
...

### Steps to reproduce (if needed)
![156](https://user-images.githubusercontent.com/8122371/42366485-e5d34b16-8133-11e8-997d-3c874eb19735.jpg)

### Environment
AppCode 2018.1.4
Build #OC-181.5087.34, built on May 23, 2018
Subscription is active until June 25, 2019
JRE: 1.8.0_152-release-1136-b39 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.5

![157](https://user-images.githubusercontent.com/8122371/42366608-3018b954-8134-11e8-9c4a-b03c4bd2cf78.jpg)

**Comment by DrDoa** [comment]
Actually, the text color on the choosing project window above the App icon is the same color, grey.
![158](https://user-images.githubusercontent.com/8122371/42367041-6c8a03e2-8135-11e8-99e5-5c439d424cc4.jpg)

**Comment by mallowigi** [comment]
weird... here is what i got:

Dark Custom
![image](https://user-images.githubusercontent.com/5015756/42375773-56ff131a-8125-11e8-930f-64c21a58bfcb.png)

Light Custom
![image](https://user-images.githubusercontent.com/5015756/42375811-746b72cc-8125-11e8-85f3-e1d7feb75266.png)

Can you provide me your settings?

**Comment by DrDoa** [comment]
its color is the same as your Dark Custom, gray. Can the color be custom myself?

**Comment by mallowigi** [comment]
You’re talking about the screenshot inside the tip of the day? It’s an image, it’s not modifiable.

**Comment by DrDoa** [comment]
Not the image,  the text in Dark Custom,Gray,  Light Custom,Blue

**Comment by mallowigi** [comment]
It’s the foreground color

**Comment by DrDoa** [comment]
I guess it is not. Because my foreground color is #009C37. As you can see in the picture.

here is my settings.

[settings.jar.zip](https://github.com/ChrisRM/material-theme-jetbrains/files/2194602/settings.jar.zip)

**Comment by DrDoa** [comment]
BTW, can I custom the icons by myself in the preference panel?

**Comment by mallowigi** [comment]
Yes i saw in the screenshot, hence why i said its weird. Maybe it didnt work and got fixed with the latest refactorings? Can you try again on an EAP version?

**Comment by mallowigi** [comment]
AppCode 2018.2 RC is out.

As for the icons, I planned to do this in the beginning but since Jetbrains is working on a solution in-house, I figured it would be pointless. However you can try to fork the project and compile the plugin with the icons you want. Take a look at http://www.material-theme.com/docs/development/installation/ for more info

**Comment by DrDoa** [comment]
EAP is much weirder.  the text in the preference is gray, too. and the color is not exactly the color I set, lighter.

**Comment by DrDoa** [comment]
Then, I feel pleased with the current version, although little promblem. The Defects did not Obscure the Virtues, great plugin thanks.
